Makefile

```diff
@@ -210,6 +210,12 @@ deploy_staging: gcloud webapp_deps package_announcer var-BRANCH_NAME var-APP_PAT
 	cd $(WPTD_PATH); util/deploy.sh -q -b $(BRANCH_NAME) $(APP_PATH)
 	rm -rf $(WPTD_PATH)revisions/service/wpt.fyi
 
+deploy_production: gcloud webapp_deps package_announcer var-BRANCH_NAME var-APP_PATH var-PROJECT $(WPTD_PATH)client-secret.json
+	gcloud config set project $(PROJECT)
+	gcloud auth activate-service-account --key-file $(WPTD_PATH)client-secret.json
```

Comment:
Nit: I *think* prod deploy will almost always be done by a human, so we probably don't need to use a service account (and don't need to depend on `client-secret.json`).